nautobot/extras/filters.py

```diff
@@ -327,6 +327,14 @@ def __init__(self, custom_field, *args, **kwargs):
             # Contains handles lists within the JSON data for multi select fields
             self.lookup_expr = "contains"
 
+    def filter(self, qs, value):
+
+        if "_custom_field_data" in self.field_name and value == "null":
```

Comment:
Is the check for `"_custom_field_data" in self.field_name` even needed here? Shouldn't it always be the case for this filter?